_data/projects.yml

```diff
@@ -215,7 +215,20 @@
   this.
   <br>More information <a href=https://github.com/ohbm/hackathon2020/issues/181>in this github issue</a>"
   hub: "Europe / Middle-East / Africa"
-
+-
+  title: "BIDS Specification Terms PDF Table Generator"
+  image: BIDStablegenerator.png
```

Comment:
Do you think we could use this image for this project?

https://github.com/Remi-Gau/OHBM_hackathon2020/blob/remi-traintrack/img/traintrack/BIDS_logo.jpg

Reply:
Check :)

Reply:
Great. Thanks. :-)